Zend/tests/return_types/026.phpt

```diff
@@ -0,0 +1,9 @@
+--TEST--
+Return type of parent is not allowed in function
+--FILE--
+<?php
+
+function test(): parent {}
+
+--EXPECTF--
+Fatal error: Cannot declare a return type of parent outside of a class scope in %s on line 3
```

Comment:
Yes, the user land test cases didn't cover the parent case, so a forget to implement that.